centinel/client.py

```diff
@@ -37,7 +37,7 @@ def get_input_file(self, experiment_name):
         return os.path.join(self.config['dirs']['data_dir'], input_file)
 
     def load_input_file(self, name):
-        input_file = self.get_input_file(name)
+        input_file = self.get_input_file("%s.txt" %(name))
```

Comment:
If you change the input files from say "world.txt" to "world", it works, so the code *is* functional. I found out shortly after getting to Toronto that the Citizen Lab had to strip all of the .txt extensions to make this work for their tests, so I was aware that this was a problem. However, we are not going to make significant changes to the functionality of a production system without some value added. It adds no value to switch everything to the .txt functionality, so I'm not going to merge this commit as is. If your other commit is urgent, you can split it into 2 pull requests and I will merge the HTTP lib stuff.

Moving forward, we should support arbitrary extensions for this particular case. This functionality is designed to match experiments with their input files, so matching against more types better fits expected functionality. Eventually, we should match against everything, so that is the fix for this problem. the glob module should let you do this and there are other examples from the client code and the server code on how to make it extension agnostic.